### PR TITLE
fixing security requeriments

### PR DESCRIPTION
**Pull request by gracekarina**
This PR fixes issue #562

### CODE REVIEW

File: modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java [hunk 1]
```diff
@@ -49,6 +49,47 @@
...
+                "info:\n" +
+                "  title: Test OpenApi Specification\n" +
+                "  version: 1.0.0\n" +
+                "paths:\n" +
+                "  /test:\n" +
+                "    get:\n" +
+                "      operationId: testGet\n" +
+                "      security:\n" +
+                "      - openId:\n" +
+                "        - https://scopes.example.com/myScope\n" +
```
Comment (vinscom) [inline modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java]: As scope is used for access control of API. Above scope is not good example of "scope". 
You can take sample value from original Open API 3 spec. 
For example:
```yaml
openId_auth:
- write:pets
- read:pets
```
or
```
openId_auth:
- write
- read
```

File: modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java [hunk 2]
```diff
@@ -2368,27 +2368,38 @@ public Operation getOperation(ObjectNode obj, String location, ParseResult resul
...
-        for (JsonNode node : nodes) {
-            if (node.getNodeType().equals(JsonNodeType.OBJECT)) {
-                SecurityRequirement securityRequirement = new SecurityRequirement();
-                Set<String> keys = getKeys((ObjectNode) node);
                 for (String key : keys) {
-                    if (key != null) {
-                        securityRequirement.addList(key,node.textValue());
-                        if (securityRequirement != null && securityRequirement.size() > 0){
-                            securityRequirements.add(securityRequirement);
+                    List<String> scopes = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (gracekarina) [reply modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java]: The original code was not loading the content of the security array.

[the remark this answers is not in the document]
Comment (gracekarina) [reply modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/OpenAPIDeserializer.java]: Ok. I see your point. thanks!

File: modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java (hunk 1 above)
Comment (gracekarina) [reply modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java]: I also added a test following this advice. Thanks!

Comment (vinscom) [inline modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java]: Great, just one last question. Why not updated "https://scopes.example.com/myScope" to reflect real world scope example. :)

Comment (vinscom) [reply modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java]: Great, just one last question. Why not updated "https://scopes.example.com/myScope" to reflect real world scope example. :)

**Comment by vinscom** [comment]
Everything looks good to me.

File: modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java (hunk 1 above)
Comment (gracekarina) [reply modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/util/OpenAPIDeserializerTest.java]: that's actually a real world example jeje. from this ticket https://github.com/swagger-api/swagger-parser/issues/562 :)

**Pull request merged**